src/doc/src/reference/unstable.md

```diff
@@ -498,10 +498,10 @@ are all finished. Hover the mouse over a unit to highlight the lines. This can
 help visualize the critical path of dependencies. This may change between runs
 because the units may finish in different orders.
 
-The "codegen" times are highlighted in a lavender color. In some cases, build
-pipelining allows units to start when their dependencies are performing code
-generation. This information is not always displayed (for example, binary
-units do not show when code generation starts).
```

Comment:
I think this is still accurate because some compliations are inherently non-pipeline-able such as binaries-after-libraries (at least until rustc gets an option to perform only the linking step)

Reply:
> at least until rustc gets an option to perform only the linking step

Technically there already is. You can split compilation and linking using `-Zno-link` and `-Zlink-only`.